docs/lang/articles/contribution/contributor_guide.md

```diff
@@ -4,296 +4,324 @@ sidebar_position: 1
-  follow.
+Check out more tags in [misc/prtags.json](https://github.com/taichi-dev/taichi/blob/master/misc/prtags.json). When introducing a new tag, please update [misc/prtags.json](https://github.com/taichi-dev/taichi/blob/master/misc/prtags.json) in the first PR with that tag so that others can follow.
 
 :::note
 
-We do appreciate all kinds of contributions, yet we should not expose
-the title of every PR to end-users. Therefore the changelog will
-distinguish *what the user should know* from *what the
-developers are doing*. This is done by **capitalizing PR
-tags**:
-
-- PRs with visible or notable features to the users should be marked
-  with tags starting with **the first letter capitalized**, e.g.,
-  `[Metal]`, `[Vulkan]`, `[IR]`, `[Lang]`, `[CLI]`. These PRs will be
-  [highlighted in the release note](https://github.com/taichi-dev/taichi/blob/master/misc/make_changelog.py)
-  for end-users, therefore it is important to make sure your PR title is
-  effective in telling what your PR does.
-- Other PRs (underlying development or intermediate implementation)
-  should use tags with **everything in lowercase letters**, e.g.,
-  `[metal]`, `[vulkan]`, `[ir]`, `[lang]`, `[cli]`.
-- Because of the way the release changelog is generated, there
-  should be **at most one capitalized tag** in a PR title to prevent
-  duplicate PR highlights. For example,
-  `[GUI] [Mac] Support modifier keys` ([#1189](https://github.com/taichi-dev/taichi/pull/1189))
-  is an improper tag choice, and we
-  should have used `[gui] [Mac] Support modifier keys in GUI` instead.
-  Please capitalize the tag that is the *most* relevant to the PR.
+We do appreciate all kinds of contributions, yet we do not intend to expose the details of every PR to the end-users. Therefore the changelog helps distinguish *what the user should know* from *what the*
+
+*developers are doing*. This is done by **capitalizing PR tags**:
+
+- PRs with visible or notable features to the users should be marked with tags starting with **a capitalized initial**, for example`[Metal]`, `[Vulkan]`, `[IR]`, `[Lang]`. These PRs are [highlighted in the release note](https://github.com/taichi-dev/taichi/blob/master/misc/make_changelog.py) for the end users, therefore it is important that your PR title be self-explanatory.
+
+- Other PRs (underlying development or intermediate implementation) should use tags **in lowercase letters**, for example`[metal]`, `[vulkan]`, `[ir]`, `[lang]`.
+
+- Because of the way the release changelog is generated, there should be **at most one capitalized tag** in a PR title to prevent duplicate PR highlights. For example, `[GUI] [Mac] Support modifier keys` ([#1189](https://github.com/taichi-dev/taichi/pull/1189))is an improper tag choice, and we should have used `[gui] [Mac] Support modifier keys in GUI` instead.
+
+Please capitalize the most relevant tag to your PR.
+
 :::
 
-## Testing
+### **Use continuous integration**
```

Comment:
CI is set up by ourselves and runs automatically on each commit. In most cases contributors wouldn't need to worry about it.

```suggestion
### Continuous Integration (CI)
```

Reply:
Agreed.